forwards_paper.tex

```diff
@@ -526,20 +548,62 @@ \subsection*{Tree sequence simplification}
 (These lineages are the ``colors'' above.)
 We also maintain a map $A$ such that $A_j$ is segment chain for node $j$ in
 the input tree sequence.
-Crucially, the time required to run the algorithm is linear in the number of edges of the input tree sequence..
+Crucially, the time required to run the algorithm is linear in the number of edges of the input tree sequence.
+
+\subsection*{Forward simulation and sequential simplification to record simplifed tree sequences}
 
-\paragraph{Sequential simplification}
-At any point the simulation scheme above, genealogical history is recorded
-in a tree sequence. This has two additional advantages. First, simplification
+At any point in Algorithm~\algref{W}, genealogical history is recorded
+in an unsimplified tree sequence. This has two additional advantages. First, simplification
```

Comment:
the point is that it is a tree sequence, so we can use our tools, not that it is unsimplified, I think.

Reply:
:+1: